chromium_src/chrome/browser/bitmap_fetcher/bitmap_fetcher_service.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../chrome/browser/bitmap_fetcher/bitmap_fetcher_service.cc"
+
+BitmapFetcherService::RequestId BitmapFetcherService::RequestImage(
```

Comment:
what are we doing here? Why do we need this?

Reply:
This is to be able to pass in our traffic annotation, otherwise the built-in Omnibox annotation will be used (https://cs.chromium.org/chromium/src/chrome/browser/bitmap_fetcher/bitmap_fetcher_service.cc?l=37)

Reply:
I don't understand why they would tie it to the omnibox like that and I don't think the traffic annotation really matters that much, but I guess this doesn't add any patches so it's fine for now. If in the future this changes in a way that does require patching then I think we should just drop the annotation. It was only added in the first place because it was required for the api